### PR TITLE
test(ui): Add Percy snapshots for releases v2

### PR DESCRIPTION
**Pull request by matejminar**
Add Percy snapshots for releases v2.
We will fill this with health data in a later PR.

### CODE REVIEW

File: src/sentry/testutils/factories.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def create_project_key(project):
         return project.key_set.get_or_create()[0]
 
     @staticmethod
-    def create_release(project, user=None, version=None, date_added=None):
+    def create_release(project, user=None, version=None, date_added=None, project2=None):
```
Comment (billyvg) [inline src/sentry/testutils/factories.py:306]: Hmm, I would make this a list of `additional_projects`

Comment (matejminar) [reply src/sentry/testutils/factories.py:306]: me too in javascript 😅need to look it up how to do that in python

Comment (matejminar) [reply src/sentry/testutils/factories.py]: wasn't that hard 👍

File: tests/acceptance/test_organization_releases_v2.py [hunk 2]
```diff
@@ -25,9 +29,37 @@ def test_no_access(self):
...
+
+    def test_detail_global_header(self):
+        with self.feature(FEATURE_NAME):
+            release = self.create_release(
+                project=self.project, additional_projects=[self.project2], version="1.0"
+            )
+            self.browser.get(u"{}?project={}".format(self.path + release.version, self.project.id))
+            self.browser.wait_until_not(".loading")
+            self.browser.click('[data-test-id="global-header-project-selector"]')
+            self.browser.snapshot("organization releases v2 - detail - global project header")
```
Comment (billyvg) [inline tests/acceptance/test_organization_releases_v2.py:65]: Do we need snapshots here? If we have snapshots elsewhere I would just make some assertions that content is there.

**Review by billyvg — APPROVED** [review]

File: tests/acceptance/test_organization_releases_v2.py (hunk 2 above)
Comment (matejminar) [reply tests/acceptance/test_organization_releases_v2.py:65]: This is snapshotting that new feature of globalSelectionHeader project picker where we see only specified projects and a custom footer message saying `Only projects with this release are visible.` I promised this to @markstory a long time ago.

Comment (billyvg) [reply tests/acceptance/test_organization_releases_v2.py:65]: 👍 I would add ^ description somewhere :)

**Review by markstory — COMMENTED** [review]
Nice work. Great to see acceptance tests/visual regressions for new features.

**Pull request merged**